src/xunit.performance.analysis/Program.cs

```diff
@@ -454,6 +310,23 @@ private static IEnumerable<TestIterationResult> ParseOneXmlFile(string path)
                 var runId = perfElem.Attribute("runid").Value;
                 var etlPath = perfElem.Attribute("etl").Value;
 
+                foreach(var metrics in perfElem.Descendants( "metrics" ))
+                {
+                    foreach(var metric in metrics.Elements())
+                    {
+                        var metricName = metric.Name.LocalName;
+                        var unit       = metric.Attribute("unit").Value;
+
+                        //
+                        // Populate the set of all collected metrics 
+                        //
+                        if(Properties.AllMetrics.ContainsKey( metricName ) == false)
```

Comment:
Should I assume that a coding convention? '== false' is more readable for me, but consistency is a value, if indeed we have guidance at that level. Is there any enforcing tool we are using in the team?

---
In reply to: [63916980](https://github.com/Microsoft/xunit-performance/pull/144#discussion_r63916980) [](ancestors = 63916980)